### ISSUE TITLE
Trouble reproducing GNN results on LBA using provided code

### ISSUE THREAD

**Issue by Calvin-Qiu** [drorlab/atom3d#45]
Hi, I'm unable to reproduce the results quoted in the paper for the performance achieved by the baseline GNN model on the LBA dataset with 30% sequence identity split. I'm following atom3d/examples/lba/gnn to download the dataset and train the model with hyperparameters given in the README. Over 6 runs with different seeds, I'm getting 1.58 +- 0.04 for test RMSE, 0.53 +- 0.03 for test Pearson and 0.54 +- 0.04 for test Spearman. Only the RMSE is consistent with what's reported in the paper. Can the authors confirm that the hyperparameters and code given in atom3d/examples/lba/gnn are identical to what's used to produce the results in the paper? Thanks!

**Comment by swenson-nick** [comment]
Bump. The results reported in the paper seem to correspond to the Validations set.

**Comment by awfderry** [comment]
Thanks for letting us know, we are looking into it and will get back to you as soon as possible!

**Comment by awfderry** [comment]
We've investigated the issue and found a bug in the GNN data processing code which resulted in data leakage between validation and test sets. The code and benchmark values in the paper have been updated. Thank you again for bringing this to our attention!

**Comment by Calvin-Qiu** [comment]
Would you please include a link to the updated paper? The paper referred to in master/README.md does not seem to have updated values on the LBA benchmark.

**Comment by martinvoegele** [comment]
I have submitted the new version to the arXiv on Saturday but it takes a bit until it takes effect. You can just follow the same link again in a few days; it should automatically lead to the latest version (then v4).

**Comment by martinvoegele** [comment]
If you need it earlier, shoot me an e-mail and I will send it to you as a PDF file.

**Comment by martinvoegele** [comment]
The new version has just been made public. It is now available at:
http://arxiv.org/abs/2012.04035
When you view the page and PDF again you may need to refresh your browser to see the updates.

**Comment by swenson-nick** [comment]
Hi, sorry to bring this up again, but I'm having similar issues with LEP. For the validation set, I'm getting around the number originally reported of 0.74 AUC-ROC, however, for the test set, I'm getting results closer to what was reported for the 3D-CNN of 0.59. Are the default parameters in /examples/lep/gnn/train.py the same ones that were used to train the models?

**Comment by awfderry** [comment]
We updated the LEP numbers as well in the new version. (now our performance is 0.681 +/- 0.062). Are you using the latest version of the code (especially train.py and data.py)?

**Comment by swenson-nick** [comment]
Thanks Alex! A standard deviation of 0.062 (almost 10%) seems very large, especially when the average is only computed over 3 trials. I also notice that the std is even larger for AUPR (over 20% of the mean).
